utils/metrics/metrics_util.go

```diff
@@ -316,3 +318,18 @@ func IncrConsumerEventCount(msgType string) {
 		[]metrics.Label{telemetry.NewLabel("msg_type", msgType)},
 	)
 }
+
+// Gauge for gas price paid for transactions
+// Metric Name:
+//
+// sei_evm_effective_gas_price
+func GaugeEvmEffectiveGasPrice(gasPrice *big.Int, blockNum uint64, txHash common.Hash) {
+	telemetry.SetGaugeWithLabels(
+		[]string{"sei", "evm", "effective", "gas", "price"},
+		float32(gasPrice.Uint64()),
+		[]metrics.Label{
+			telemetry.NewLabel("block_num", strconv.FormatUint(blockNum, 10)),
+			telemetry.NewLabel("tx_hash", txHash.Hex()),
```

Comment:
we probably don't want tx_hash as a label because it's so high-cardinality and hard to aggregate